pages/linux/rpicam-vid.md

```diff
@@ -1,13 +1,8 @@
 # rpicam-vid
 
 > Capture a video using a Raspberry Pi camera.
-> Some subcommands such as `vlc` have their own usage documentation.
 > More information: <https://www.raspberrypi.com/documentation/computers/camera_software.html#rpicam-vid>.
```

Comment:
```suggestion
> See also: `vlc`.
> More information: <https://www.raspberrypi.com/documentation/computers/camera_software.html#rpicam-vid>.
```

Reply:
> You can use ``See also: `command1`, `command2`.`` template to reference similar commands, aliases and subcommands.

Is VLC related in any way to `rpicam-vid`? You can play the recorded videos with any program you want.